crates/forge/tests/it/test_helpers.rs

```diff
@@ -159,6 +184,43 @@ impl ForgeTestProfile {
 
         config
     }
+
+    /// Build [Config] for zksync test profile.
+    ///
+    /// Project source files are read from testdata/zk
+    /// Project output files are written to testdata/zk/out and testdata/zk/zkout
+    /// Cache is written to testdata/zk/cache
+    ///
+    /// AST output is enabled by default to support inline configs.
+    pub fn zk_config(&self) -> Config {
+        let mut zk_config = Config::with_root(self.root());
+
+        zk_config.ast = true;
+        zk_config.src = self.root().join("./zk");
+        zk_config.test = self.root().join("./zk");
+        zk_config.out = self.root().join("zk").join("out");
+        zk_config.cache_path = self.root().join("zk").join("cache");
+        zk_config.libraries = vec![
+            "fork/Fork.t.sol:DssExecLib:0xfD88CeE74f7D78697775aBDAE53f9Da1559728E4".to_string(),
```

Comment:
Do we have this "predeployed" to the in memory node? Or otherwise a deterministic deployment?

Reply:
Probably not needed, carried over from foundry test. Removed.